src/app/cube/new-home/help/components/help-card/help-card.component.html

```diff
@@ -1,8 +1,8 @@
-<div class="helpCard">
+<div class="helpCard" [routerLink]="option.link">
     <span [ngClass]="option.iconColor">
         <i [ngClass]="option.icon"></i>
     </span>
     <h2>{{ option.title }}</h2>
     <p>{{ option.description }}</p>
-    <a>Get Started <i class="fa fa-angle-right" aria-hidden="true"></i></a>
+    <a [routerLink]="option.link">Get Started <i class="fa fa-angle-right" aria-hidden="true"></i></a>
```

Comment:
what happens when option.link is undefined since its an optional field?

Reply:
The component doesn't link anywhere and Emily's template implementation still works.

Reply:
Is there ever a case where option could be undefined?

Reply:
all of the options are hard-coded in the help component and emily used an ngFor to iterate over them, unless we add an empty object to the array then I don't think so.